frame/scheduler/src/lib.rs

```diff
@@ -260,6 +262,55 @@ decl_module! {
 			Self::do_cancel_named(Some(origin.caller().clone()), id)?;
 		}
 
+		/// Anonymously schedule a task after a delay.
+		///
+		/// # <weight>
+		/// - S = Number of already scheduled calls
+		/// - Base Weight: 22.29 + .126 * S µs
+		/// - DB Weight:
+		///     - Read: Agenda
+		///     - Write: Agenda
+		/// - Will use base weight of 25 which should be good for up to 30 scheduled calls
+		/// # </weight>
+		#[weight = 25_000_000 + T::DbWeight::get().reads_writes(1, 1)]
+		fn schedule_after(origin,
+			after: T::BlockNumber,
+			maybe_periodic: Option<schedule::Period<T::BlockNumber>>,
+			priority: schedule::Priority,
+			call: Box<<T as Trait>::Call>,
+		) {
+			T::ScheduleOrigin::ensure_origin(origin.clone())?;
+			let origin = <T as Trait>::Origin::from(origin);
+			Self::do_schedule(
+				DelayedDispatchTime::After(after), maybe_periodic, priority, origin.caller().clone(), *call
+			)?;
+		}
+
+		/// Schedule a named task after a delay.
+		///
+		/// # <weight>
+		/// - S = Number of already scheduled calls
```

Comment:
Since I think you haven't benchmarked these and you are using the weight details of `schedule_named ` and `schedule`, I would just say: 
```suggestion
Same as [`schedule_named`]. 
```